content/posts/writing-posts/rich-content.md

```diff
@@ -16,7 +16,7 @@ Hugo ships with several [Built-in Shortcodes](https://gohugo.io/content-manageme
 
 ## Instagram Simple Shortcode
 
-{{< instagram_simple BGvuInzyFAe hidecaption >}}
+<!-- https://github.com/chipzoller/hugo-clarity/issues/69 -->
```

Comment:
This is giving me a build error

Reply:
The internal template will updated soon. We can comment this whole instragram section (including title) till then.